src/libponyrt/sched/start.c

```diff
@@ -156,7 +156,14 @@ static int parse_opts(int argc, char** argv, options_t* opt)
       case OPT_VERSION: opt->version = true; break;
       case OPT_PONYHELP: opt->ponyhelp = true; break;
 
-      default: exit(-1);
+      case -2:
```

Comment:
can we give this a meaningful name via a `#define`. Something along the lines of `OPT_ERROR`?

Reply:
I might come back to this as I think the whole options processing code would benefit from some simplifications.